app/src/main/java/pl/ppiwd/exerciseanalyst/utils/Vector3d.java

```diff
@@ -0,0 +1,13 @@
+package pl.ppiwd.exerciseanalyst.utils;
+
+public class Vector3d {
+    public float x;
```

Comment:
Set to private and add getters for each field.

Reply:
Must be public to work with Room Database. This class is POJO

Reply:
Ok, It turns out that when set of getters and setters are provided for those classes used by Room Database then everything is fine. I've changed fields to be private and generated getters/setters for them.
Thanks for pointing it out!

Reply:
I believe that getters and all args constructor also would be fine and it would be more OOP, but I'm fine with setters.

Reply:
I checked that, Framework was complaining about missing setters

Reply:
IMHO unless the setter provides more value than just an assignment, it's not really needed. Same goes for the getter, it's just forwarding the value to the higher scope. The initial - all public, no get/set function implementation was good. Feel free to leave it as it though.